src/App.js

```diff
@@ -1,23 +1,63 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import React, { useState, useEffect } from "react";
+import Icon from "./components/Icon";
+
+const getQueryUrl = (city) =>
+  `${process.env.REACT_APP_API_URL}?q=${city}&units=metric&appid=${process.env.REACT_APP_API_KEY}`;
 
 function App() {
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState("");
+  const [weatherData, setWeatherData] = useState();
+  const [cityInput, setCityInput] = useState("Warszawa");
+  const [city, setCity] = useState(cityInput);
```

Comment:
@pasternakewa WOT? why You need this state?

Reply:
To fetch only on input submit, not on every input change